stl/inc/vector

```diff
@@ -790,14 +668,18 @@ public:
         _Constructed_first = _Newvec + _Whereoff;
 
         if (_Whereptr == _Mylast) { // at back, provide strong guarantee
-            _Umove_if_noexcept(_Myfirst, _Mylast, _Newvec);
+            if constexpr (is_nothrow_move_constructible_v<_Ty> || !is_copy_constructible_v<_Ty>) {
+                _Uninitialized_move(_Myfirst, _Mylast, _Newvec, _Getal());

```

Comment:
We already called _Getal() at the beginning of the function, can we use it again here?

Below as well x4